app/controllers/shipments_controller.rb

```diff
@@ -1,5 +1,17 @@
+require 'dotenv/load'
+require 'aftership'
+require 'json'
+
 class ShipmentsController < ApplicationController
   def index
     @shipments = Shipment.all
   end
+
+  def show
+    @company = Company.find(params[:company_id])
+    @shipment = @company.shipments.find(params[:id])
+
+    AfterShip.api_key = ENV['AFTERSHIP_KEY']
+    @tracker = AfterShip::V4::Tracking.get(@shipment.slug, @shipment.tracking_number)
```

Comment:
> The returned JSon data should be displayed on the show page but for some reason I get the following error.
undefined method `status' for #Hash:0x000055b4643f4328

Were you able to see what the Aftership response was? What slug and tracking number were you testing with?

Reply:
Apologies I only just saw these messages. Yes the response I am receiving is currently null. I am thinking this is due to using a fake tracking number from the seed file that came with the test. Would this be correct?